### ISSUE TITLE
Metadata of RobotFramework_TestsuitesManagement (1)

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/robotframework-testsuitesmanagement#102]
Without calling the `testsuite_setup` of `RobotFramework_TestsuitesManagement` and without defining `Metadata` inside the robot file, the `SUITE_METADATA` dictionary of Robot Framework is empty.

With metadata defined in robot file, e.g. like this:
`Metadata    my_own_metadata    my_own_metadata_value`
the `SUITE_METADATA` dictionary contains this value.

Calling additionally the `testsuite_setup` of `RobotFramework_TestsuitesManagement` causes additional metadata values. This is indicated in log file by entries like tat:
```
Set suite metadata 'project' to value 'RobotFramework Testsuites'.
Set suite metadata 'version_sw' to value ''.
Set suite metadata 'version_hw' to value ''.
Set suite metadata 'version_test' to value ''.
Set suite metadata 'machine' to value '...'.
Set suite metadata 'tester' to value '...'.
Set suite metadata 'testtool' to value 'Robot Framework 4.1.3 (Python 3.9.16 on win32)'.
Set suite metadata 'version' to value '0.6.0'.
```

This listing does not contain metadata defined in robot files. This is fine to me because I accept that these lines are generated by a separate mechanism. But this is also mistakable. A not so much experienced user might not be aware of this, and this user might ask: Where are the other metadata values?

I would prefer the log files to point out more strict from where the values come from, e.g. like this:

```
Metadata added by RobotFramework_TestsuitesManagement:
Set suite metadata 'project' to value 'RobotFramework Testsuites'.
...
```

In this case it becomes more obvious that there might be other instances involved who also add values to the metadata.

**Comment by test-fullautomation** [comment]
Hi Son,
please implement accordingly.
Thank you,
 Thomas

**Comment by namsonx** [comment]
Hello Thomas,

I put the comment in ticket #103, in case you agree with the proposal of I and Holger then I will implement this ticket and #103 in one new pull-request.

Thank you,
Son

**Comment by test-fullautomation** [comment]
Hi Son,

O.K. for me.

Thank you,
 Thomas

**Comment by HolQue** [comment]
Hi Son, hi Thomas,

please urgently consider also my latest comment at end of:

https://github.com/test-fullautomation/robotframework-testsuitesmanagement/issues/103#issuecomment-1431245925

It's partially in opposite to the suggestions in this issue. Before implementation starts we need a common agreement about how this issue shall be handled.

**Comment by namsonx** [comment]
Hello Thomas,

I created the pull-request #126  for this ticket.

Thank you,
Son

**Comment by test-fullautomation** [comment]
solved and already merged with #123
